src/integrationCapture.ts

```diff
@@ -221,8 +227,9 @@ export default class IntegrationCapture {
      * Captures local storage based on the integration ID mapping.
      */
     public captureLocalStorage(): Dictionary<string> {
+        const keys = this.getAllowedKeysForMode();
```

Comment:
Maybe we be more descriptive here.  Is this more accurate?
```suggestion
        const integrationKeys = this.getAllowedKeysForMode();
```

Reply:
I see `activeKeys` below.  I'd just like to avoid using keys because it's too generic.  This is a non-blocking comment btw

Reply:
Replaced `keys` and `activeKeys` with `integrationKeys`